cmd/lilypad/solver.go

```diff
@@ -34,12 +35,18 @@ func newSolverCmd() *cobra.Command {
 	return solverCmd
 }
 
-func runSolver(cmd *cobra.Command, options solver.SolverOptions) error {
+func runSolver(cmd *cobra.Command, options solver.SolverOptions, network string) error {
 	commandCtx := system.NewCommandContext(cmd)
 	defer commandCtx.Cleanup()
 
-	noopTracer := noop.NewTracerProvider().Tracer(system.GetOTelServiceName(system.SolverService))
-	web3SDK, err := web3.NewContractSDK(commandCtx.Ctx, options.Web3, noopTracer)
+	telemetry, err := configureTelemetry(commandCtx.Ctx, system.SolverService, network, options.Telemetry, options.Web3)
+	if err != nil {
+		fmt.Printf("failed to setup opentelemetry: %s", err)
```

Comment:
why aren't we using `log` here?

Reply:
Same nit from me too but otherwise looks good

Reply:
No good reason for it. Updated it to a warn log, and also updated similar in the resource provider:

https://github.com/Lilypad-Tech/lilypad/blob/224ee5bf4fd59b6fae638973b479778ef0f67612/cmd/lilypad/resource-provider.go#L43